src/main/java/application/Routes.java

```diff
@@ -0,0 +1,37 @@
+package main.java.application;
+
+import main.java.server.Router;
+
+import java.util.HashMap;
+import java.util.TreeMap;
+
+public class Routes {
+    private static TreeMap<String, HashMap<String, Handler>> ROUTES;
+    static {
+        Router router = new Router();
+        router.get("/simple_get", Handlers.SimpleGet);
+        router.head("/simple_get", Handlers.SimpleGet);
+        router.options("/simple_get", Handlers.SimpleOptions);
+
+        router.head("/get_with_body", Handlers.GetWithBody);
+        router.options("/get_with_body", Handlers.GetWithBody);
+
+        router.options("/method_options", Handlers.SimpleOptions);
+        router.get("/method_options", Handlers.SimpleGet);
+        router.head("/method_options", Handlers.SimpleGet);
+
+        router.get("/method_options2", Handlers.SimpleGet);
+        router.options("/method_options2", Handlers.SimpleOptions);
+        router.head("/method_options2", Handlers.SimpleGet);
+        router.put("/method_options2", Handlers.EchoBody);
+        router.post("/method_options2", Handlers.EchoBody);
+
+        router.post("/echo_body", Handlers.EchoBody);
+
+        ROUTES = router.routes();
+    }
+
+    public static TreeMap<String, HashMap<String, Handler>> getROUTES() {
+        return ROUTES;
```

Comment:
I'm all about getters and setters, but perhaps we can get away with just using this value directly if it's a constant defined at build time?